django_project/frontend/utils/country_level_dark_v1.json

```diff
@@ -0,0 +1,583 @@
+{
+  "version": 8,
+  "name": "sanbi",
+  "metadata": {
+    "maputnik:license": "https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md",
+    "maputnik:renderer": "mbgljs",
+    "openmaptiles:version": "3.x",
+    "sanbi:version": "12",
+    "sanbi:theme": "dark"
+  },
+  "center": [25.86, -28.52],
+  "zoom": 5,
+  "sources": {
+    "sanbi": {
+      "type": "vector",
+      "tiles": ["http://localhost:9090/maps/sanbi/{z}/{x}/{y}.pbf"],
+      "minZoom": 0,
```

Comment:
I think let's set to minZoom: 5. This is one of the change that I need to do for importing Jeremy style.
Can also change other occurrance for minZoom: 0?
Please also apply to light theme